docs/examples/userguide/language_basics/casting_python.py

```diff
@@ -8,7 +8,7 @@ def main():
     # as the correct pointer type that is assigned to them.
     # They do not need to be typed explicitly.
 
-    ptr = cython.cast(cython.p_void, python_string)
+    ptr = cython.cast(cython.pointer(void), python_string)
```

Comment:
Does this not work? I'm fairly sure `p_void` should be valid.

Reply:
I was not sure! Thanks for the review, so I updated it to `p_void`.